pam_mysql.c

```diff
@@ -632,9 +636,66 @@ static char *pam_mysql_md5_data(const unsigned char *d, unsigned int sz, char *m
 #endif
 /* }}} */
 
-/* {{{ pam_mysql_sha1_data */
 #if defined(HAVE_OPENSSL)
 #define HAVE_PAM_MYSQL_SHA1_DATA
+/* {{{ calcDecodeLength */
```

Comment:
Could I get you to put in 'proper' documentation? (Ala PHPdocs / kernel docs) I've started to do this for other functions in the file as you'll see from last night (Australian time)'s commits.